doc/configuration.rst

```diff
@@ -74,20 +74,21 @@ Kansha features can be activated and customized with a configuration file like t
     key = <<AUTH_OAUTH_KEY>>
     secret = <<AUTH_OAUTH_SECRET>>
 
+    [locale]
+    major = fr
+    minor = FR
 
-    [mail]
+    [services]
+    [[mail]]
     activated = on
     smtp_host = <<MAIL_HOST>>
     smtp_port = <<MAIL_PORT>>
```

Comment:
'smtp_host' -> 'host' / 'smtp_port' -> 'port' because we already are in the 'mail_sender' section